dandi/tests/data/metadata/metadata2asset_3.json

```
@@ -81,6 +81,9 @@
         "identifier": "http://purl.obolibrary.org/obo/PATO_0000384",
         "name": "Male"
       },
+      "strain": {
+        "name": "abcdef/1"
```

```suggestion
        "name": "abcdef/1",
        "schemaKey": "StrainType"
```

seems needed according to failing tests

---

Let me just try ;) committing